src/modules/Exchange.ts

```diff
@@ -1,13 +1,15 @@
 import Big from 'big.js';
 import BigNumber from 'bignumber.js';
+import { at } from 'lodash';
```

Comment:
Style: prefer import _ from 'lodash', otherwise I find it confusing to have all these random names like "at", "pick" etc in scope


This is my recommendation but if you or someone has expressed a different preference lmk